src/pages/pageIndex.re

```diff
@@ -1,10 +1,13 @@
+open Util;
+
 let component = ReasonReact.statelessComponent("Home");
 
 module Link = Gatsby.Link;
 
 module Tickets = Tito.Tickets;
 
-let ste = ReasonReact.stringToElement;
+let speakerColumn = (i, speaker) =>
+  <SpeakerCard speaker key=(string_of_int(i)) />;
```

Comment:
It's a bad practice to use `index` for `key`... Should we use `name` since it's unique?